specification/datastore/DatastoreApis.ts

```diff
@@ -56,7 +56,7 @@ export const DatastoreApiSchemas = {
       adminSignature: signatureValidation
         .optional()
         .describe('A signature from the admin Identity'),
-      adminFunction: z.object({
+      adminRunner: z.object({
```

Comment:
This should still be admin"Function". Its about a js function in the datastore

Reply:
Yes, fixed.